### PR TITLE
hanoi tower - stacks

### PR DESCRIPTION
**Pull request by leonardo-anjos**
my first contribution to this repository :)

### CODE REVIEW

File: hanoi_tower/Main.c [hunk 1]
```diff
@@ -0,0 +1,198 @@
+#include <stdio.h>
```
Comment (deadshotsb) [inline hanoi_tower/Main.c:1]: @leonardo-anjos Kindly add a description and a sample case for your code.

File: hanoi_tower/Stack.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+typedef struct stack Stack;
```
Comment (kvedala) [inline hanoi_tower/Stack.h:1]: Needs #ifdef wrapper for a header file

File: hanoi_tower/Stack.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Stack *create();
+void destroy(Stack *p);
+void stackUp(Stack *p, int x); // push
+int unstack(Stack *p);         // pop
+
+int length(Stack *p);
+int top(Stack *p);
+void printStack(Stack *p);
+
+void reverse3(Stack *p);
```
Comment (kvedala) [inline hanoi_tower/Stack.h:12]: Header files must end with a newline character

File: hanoi_tower/Main.c (hunk 1 above)
Comment (leonardo-anjos) [reply hanoi_tower/Main.c:1]: ok, thanks

File: hanoi_tower/Stack.h (hunk 2 above)
Comment (leonardo-anjos) [reply hanoi_tower/Stack.h:1]: thanks

File: hanoi_tower/Stack.h (hunk 3 above)
Comment (leonardo-anjos) [reply hanoi_tower/Stack.h:12]: ok

**Pull request closed without merging**